frontend/src/components/text/TextWrapperNullableNumberEditor.tsx

```diff
@@ -21,14 +22,16 @@ export function TextWrapperNullableNumberEditor(
         </p>
       ) : (
         <div className={props.valueClassName ?? ""}>
-          {NullableNumberEditor(
-            props.value,
-            (newValue: number | undefined) =>
-              props.onValueChange(newValue ?? props.defaultValue),
-            "w-4",
-            props.disabled,
-            props.min
-          )}
+          <p className="vertical-align-middle">
```

Comment:
why is this changed to vertical-align-middle? what does it do?

Reply:
I had to make the input fields smaller so that I could fit the import field buttons, but making them smaller made them off-centered with the label. I added the vertical-align-middle to fix this